app/com/lynxanalytics/biggraph/graph_api/MetaGraph.scala

```diff
@@ -22,6 +22,24 @@ import scala.collection.immutable.SortedMap
 import com.lynxanalytics.biggraph.spark_util.SortedRDD
 import org.apache.spark
 
+object MetaGraphUtil {
+  val oldPrefix = "ACED00057372000C7363616C612E53796D626F6C292AC645426F2F4B"
+    .sliding(2, 2).map(Integer.parseInt(_, 16).toByte).toSeq
+  val newPrefix = "ACED00057372000C7363616C612E53796D626F6C5F4785C13785FF06"
```

Comment:
Maybe just overwrite the necessary section.

Reply:
> Maybe just overwrite the necessary section.

Done.